### PR TITLE
VSS to DDS-IDL exporter

### PR DESCRIPTION
**Pull request by zam1kor**
This pull request contains the code & docu to convert "VSS files to DDS IDL file"
IDL stands for Interface Definition Language
DDS stands for Data Distribution Service
Generated IDL file can be used by DDS implementations (e.g: CycloneDDS, FastDDS) to generate (python/c++) code for the elements

### CODE REVIEW

File: docs/VSS2IDL.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- DDS : Data Distribution Service
+- IDL : Interface Definition Language
+- VSS : Vehicle Signal Specification
+
+## Reference to the specifications:
+
+- VSS description can be found at : [link](https://covesa.github.io/vehicle_signal_specification/introduction/)
+- DDS  IDL specification can be found at : [link](https://www.omg.org/spec/IDL/4.2/PDF)
+
+## Various VVS elements considered for conversion:
```
[the remark this answers is not in the document]
Comment (zam1kor) [reply docs/VSS2IDL.md:13]: defined in acronyms section 
VSS : Vehicle Signal Specification

File: docs/VSS2IDL.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+| uint32  | unsigned long |
+| int32  | long           |
+| uint64  | unsigned long long|
+| int64  | long long          |
+| boolean  | boolean          |
+| float  | float              |
+| double  | double            |
+| string  | string            |
+
+## Examples of VSS data and conveted DDS-IDL data
```
[the remark this answers is not in the document]
Comment (zam1kor) [reply docs/VSS2IDL.md:43]: Thanks !! will fix it

File: docs/VSS2IDL.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (erikbosch) [reply docs/VSS2IDL.md:13]: Yes, but right now it says "VVS"

[the remark this answers is not in the document]
Comment (zam1kor) [reply docs/VSS2IDL.md:13]: got it.. overlooked at the typo :P will fix it

**Comment by danielwilms** [comment]
create PR in VSS repo (Makefile)

Decision to merge 2022/06/07

**Pull request merged**